tests/system/robot/notepadTests.robot

```diff
@@ -19,7 +19,7 @@ Test Teardown	default teardown
 default teardown
 	${screenshotName}=	create_preserved_test_output_filename	failedTest.png
 	Run Keyword If Test Failed	Take Screenshot	${screenShotName}
-	Run Keyword If Test Failed	dump_speech_to_log
+	dump_speech_to_log
```

Comment:
It was an artifact from debugging, but it can be helpful even when the tests pass and doesn't really have any drawbacks. I think it's best to leave it in.